<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1071</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_currentPosition">
       <property name="text">
        <string>Current position:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_currentPosition">
       <item>
        <widget class="QLabel" name="label_leftStation">
         <property name="text">
          <string>Concorde</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rightStation">
         <property name="text">
          <string>Tuileries</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_lineID">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Line #12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_orCoach_3">
       <item>
        <widget class="QLabel" name="label_coach_3">
         <property name="text">
          <string>Coach #3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_or_3">
         <property name="text">
          <string>98 %</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_orCoach_1">
       <item>
        <widget class="QLabel" name="label_coach_1">
         <property name="text">
          <string>Coach #1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_or_1">
         <property name="text">
          <string>85 %</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_luggageDetected">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Luggage has been detected in coach #3.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_orCoach">
       <item>
        <widget class="QLabel" name="label_orText">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Occupacy rate:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_orValue">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;85 %&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_orCoach_2">
       <item>
        <widget class="QLabel" name="label_coach_2">
         <property name="text">
          <string>Coach #2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_or_2">
         <property name="text">
          <string>50 %</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_lastMeasure">
       <property name="text">
        <string>Last measure: less than a minute ago</string>
       </property>
       <property name="buddy">
        <cstring>pushButton_measureNow</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::DaySection</enum>
       </property>
       <property name="displayFormat">
        <string>dddd dd MMMM HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="comboBox_chooseAnotherVehicle"/>
     </item>
     <item row="13" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_orCoach_4">
       <item>
        <widget class="QLabel" name="label_coach_4">
         <property name="text">
          <string>Coach #4:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_or_4">
         <property name="text">
          <string>79 %</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_orPerCoach">
       <property name="text">
        <string>Occupacy rate per coach:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_isHumaneDriven">
       <property name="font">
        <font>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>This vehicle is automatically driven.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_trainID">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Train #2241&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_selectAnotherVehicle">
       <property name="text">
        <string>Select another vehicle:</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_chooseAnotherVehicle</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushButton_measureNow">
       <property name="text">
        <string>Measure now</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="13">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_coach_1">
        <attribute name="title">
         <string>Coach #1</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>19</y>
           <width>521</width>
           <height>461</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_tabCoach_1">
          <item row="2" column="2">
           <widget class="QLabel" name="label_luggage_2_1">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_camera_3_1">
            <property name="text">
             <string>Camera #3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_luggage_3_1">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_people_3_1">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_camera_2_1">
            <property name="text">
             <string>Camera #2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_people_1_1">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_people_2_1">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_luggage_1_1">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_camera_1_1">
            <property name="text">
             <string>Camera #1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label_results_1_1">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 1 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_results_2_1">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 2 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="label_results_3_1">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 3 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_coach_2">
        <attribute name="title">
         <string>Coach #2</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>431</width>
           <height>461</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_tabCoach_2">
          <item row="2" column="2">
           <widget class="QLabel" name="label_luggage_2_2">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_camera_3_2">
            <property name="text">
             <string>Camera #3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_luggage_3_2">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_people_3_2">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_camera_2_2">
            <property name="text">
             <string>Camera #2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_people_1_2">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_people_2_2">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_luggage_1_2">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_camera_1_2">
            <property name="text">
             <string>Camera #1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label_results_1_2">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 1 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_results_2_2">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 2 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="label_results_3_2">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 3 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_coach_3">
        <attribute name="title">
         <string>Coach #3</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>431</width>
           <height>461</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_tabCoach_3">
          <item row="2" column="2">
           <widget class="QLabel" name="label_luggage_2_3">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_camera_3_3">
            <property name="text">
             <string>Camera #3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_luggage_3_3">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_people_3_3">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_camera_2_3">
            <property name="text">
             <string>Camera #2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_people_1_3">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_people_2_3">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_lugagge_1_3">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_camera_1_3">
            <property name="text">
             <string>Camera #1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label_results_1_3">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 1 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_results_2_3">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 2 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="label_results_3_3">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 3 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_coach_4">
        <attribute name="title">
         <string>Coach #4</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>431</width>
           <height>461</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_tabCoach_4">
          <item row="2" column="2">
           <widget class="QLabel" name="label_luggage_2_4">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_camera_3_4">
            <property name="text">
             <string>Camera #3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_luggage_3_4">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_people_3_4">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_camera_2_4">
            <property name="text">
             <string>Camera #2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_people_1_4">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_people_2_4">
            <property name="text">
             <string>People: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_luggage_1_4">
            <property name="text">
             <string>Luggage:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_camera_1_4">
            <property name="text">
             <string>Camera #1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLabel" name="label_results_1_4">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 1 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_results_2_4">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 2 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="label_results_3_4">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Detection results for Camera 3 will soon be displayed here.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
